src/worker/pbs_info.rs

```diff
@@ -0,0 +1,53 @@
+use std::process::Command;
+use std::{str, env};
+use regex::Regex;
+use std::time::Duration;
+
+type TimeNumberFormat = u64;
+
+fn get_value_from_qstat_string(command_output: &String, property_name: &mut String, data_pattern: &String) -> String {
```

Comment:
What is the format of the `qstat` output? Please add a documentation comment to the function so that we know how the parser behaves and what should its input/output look like.